### PR TITLE
Update pre-commit config

### PR DESCRIPTION
**Pull request by cdce8p**
## Description
Updated pre-commit hooks. Removed invalid config option for `black`.

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :bug: Bug fix  |
| ✓  | :sparkles: New feature |
|   | :hammer: Refactoring  |
|   | :scroll: Docs |

--

This should be safe to cherry-pick for `2.5.1`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,8 +22,11 @@
 from setuptools.command import easy_install  # pylint: disable=unused-import
 from setuptools.command import install_lib  # pylint: disable=unused-import
 
-if sys.version_info.major == 3 and sys.version_info.minor <=5:
-    warnings.warn("You will soon need to upgrade to python 3.6 in order to use the latest version of Astroid.", DeprecationWarning)
+if sys.version_info.major == 3 and sys.version_info.minor <= 5:
```
Comment (Pierre-Sassoulas) [inline setup.py:25]: Unrelated to the MR but I think we can now remove that ?

**Review by Pierre-Sassoulas — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (cdce8p) [reply setup.py:25]: Yeah, absolutely. Didn't even notice it while preparing the PR 😅

**Comment by cdce8p** [comment]
@Pierre-Sassoulas I would suggest to cherry-pick these commits to the `2.5` branch. Since a lot of files have changed, this will prevent further conflicts.

**Comment by Pierre-Sassoulas** [comment]
Yeah, maybe even all the master branch could be directly merged by the look of it.

**Comment by cdce8p** [comment]
That would probably work as well. Could you take a look at #914 if you have some spare time? Might be something we also want to include.

**Pull request merged**